editor/project_manager.cpp

```diff
@@ -1244,7 +1244,7 @@ ProjectManager::ProjectManager() {
 	top_hb->add_child(ccl);
 	top_hb->add_spacer();
 	l = memnew(Label);
-	l->set_text("v" VERSION_MKSTRING);
+	l->set_text("v" VERSION_MKSTRING "-" + String(VERSION_HASH).left(7));
```

Comment:
Since the string uses `.` as delimiter between all fields, I guess it might be more consistent to do the same here.